api/src/main/java/fr/catcore/server/translations/api/mixin/MixinTranslatableText.java

```diff
@@ -68,6 +71,11 @@ private MutableText selfAsLiteral(LocalizationTarget target) {
                 literal = literal.append(text);
             }
         }
+        if (literal != null) {
```

Comment:
I more meant like I'm not sure if we should be having this logic at all - it's compensating for something that should probably be dealt with at a lower level.

Reply:
where should it be then? It's a needed logic :(

Reply:
If this piece of code is not here or not at the place it was before, game will looks like this with any language other than english. ![2020-09-02_22 03 43](https://user-images.githubusercontent.com/34719527/92031032-3896a580-ed68-11ea-901a-3f4e2d5a9c6c.png)

Reply:
I feel like there is something I'm not understanding correctly about what you are saying :thinking:

Reply:
I'll have a go at implementing it :)